samcli/lib/remote_invoke/remote_invoke_executor_factory.py

```diff
@@ -142,4 +179,5 @@ def _create_lambda_boto_executor(
         ],
     ] = {
         AWS_LAMBDA_FUNCTION: _create_lambda_boto_executor,
+        AWS_STEPFUNCTIONS_STATEMACHINE: _create_stepfunctions_boto_executor,
```

Comment:
I think we are going to remove this until we release this feature, right?

Reply:
Yes that's right, added this here so that Jacob can play around with the draft PR. Can update this before merging.

Reply:
I have removed this in the new commits.